flang/include/flang/Parser/parse-tree.h

```diff
@@ -3424,7 +3424,17 @@ struct AssignedGotoStmt {
 
 WRAPPER_CLASS(PauseStmt, std::optional<StopCode>);
 
-// Parse tree nodes for OpenMP 4.5 directives and clauses
+// Parse tree nodes for OpenMP 4.5+ directives and clauses
```

Comment:
If this is a general update then may be saying OpenMP 5.2 is better.

Reply:
Done.